plugins/action_generate_review_assignment_records/src/generateReviewAssignments.test.ts

```diff
@@ -112,6 +112,7 @@ test('Test: Submit Application ID#4001 - Stage 1 (Last level)', () => {
         reviewAssignmentIds: [1005, 1062, 1063, 1064, 1065],
         reviewAssignmentAssignerJoins: [],
         reviewAssignmentAssignerJoinIds: [],
+        removedAssignmentIds: [],
```

Comment:
Not really important, but I notice some of the existing tests have a "templateId" parameter, which is not required. Might be best to get rid of it?

Reply:
Hmm. Sure. I'll take it off now : )